### PR TITLE
24.04 copy update

### PR DESCRIPTION
**Pull request by carkod**
## Done

Most 24.04 updates were reviewed and updated in https://github.com/canonical/cn.ubuntu.com/pull/720, these are just some additional updates according to copy doc.

## QA

Check https://cn-ubuntu-com-722.demos.haus/download/desktop against [copy doc comment](https://docs.google.com/document/d/1-rYu9BV4mN04yXn01QhBillOyHQOeuCbuGKiJQpJjYs/edit#heading=h.lf91i2rsa8g8)
## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-11402

### CODE REVIEW

**Comment by akbarkz** [comment]
@carkod you branch seems to be outdated from the base branch, can you rebase please?

**Comment by carkod** [comment]
@britneywwc can you review/proofread? Thanks

File: templates/download/desktop.html [hunk 1]
```diff
@@ -49,7 +49,7 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
           <a class="p-button--positive is-wide" href="https://releases.ubuntu.com/{{ releases.lts.short_version }}/ubuntu-{{ releases.lts.full_version }}-desktop-amd64.iso" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Desktop', 'eventLabel' : '{{ releases.lts.short_version }}', 'eventValue' : undefined });">下载 {{ releases.lts.full_version }}</a>
           <script>performance.mark("Download (Desktop) button rendered")</script>
         </p>
-        <p><small>如需其他版本的Ubuntu桌面，包括BT下载，网络安装器，本地镜像站和历史版本，请访问 <a href="/download/alternative-downloads">其他下载</a>。</small></p>
+        <p><small>如需其他版本的Ubuntu桌面，请参考Ubuntu桌面的<a href="https://ubuntu.com/desktop">更多信</a></small></p>
```
Comment (britneywwc) [inline templates/download/desktop.html:52]:
```suggestion
        <p><small>如需其他版本的Ubuntu，请参考Ubuntu桌面的<a href="https://ubuntu.com/desktop">更多信息</a></small></p>
```

**Review by britneywwc — CHANGES_REQUESTED** [review]

**Review by britneywwc — APPROVED** [review]
LGTM 谢谢!

**Comment by carkod** [comment]
> LGTM 谢谢!

谢谢你

**Pull request merged**